ebu/ear-production-suite
# Add Apple silicon support

* add default and packaging presets for macOS/arm64
* update WDL to latest version for compatibility
* patch JUCE for arm64
* update Catch2 to v3 and include as submodule
* update tests to use Catch2 v3 interface
* use libadm/libear branches with CMake resouce embedder
* do nng version check rather than attempting to compile a sample
